Dockerfile

```diff
@@ -11,11 +11,20 @@ RUN apt-get update \
     && apt-get -qy upgrade --fix-missing --no-install-recommends \
     && apt-get -qy install --fix-missing --no-install-recommends \
         curl \
+        git \
```

Comment:
Indeed, when I made this PR I looked at the Invenio provisioning recipes :-)

I think @switowski's question is more about the benefits of using the non-slim version of the python image versus installing the decencies we need manually.
I guess you can see it as a philosophical question